x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -214,10 +214,10 @@ describe('Task Runner', () => {
     expect(call.rule.consumer).toBe('bar');
     expect(call.rule.enabled).toBe(true);
     expect(call.rule.schedule).toMatchInlineSnapshot(`
-    Object {
```

Comment:
for this file, you'll want to hide whitespace changes, will reduce the amount of code to review :-)